Automattic/newspack-block-theme
# fix(buttons): outline variations missing style with wp 6.6

### All Submissions:






### Changes proposed in this Pull Request:

With WP 6.6, the outline button styles disappeared in the editor. For some reason, the styles applied via theme.json aren't being displayed in the back-end. This PR adds the background/border/color default properties to the scss file.

There's also a small fix for the donate block so the outline buttons on `:hover` matches  the style of this theme.

Closes #217 .

### How to test the changes in this Pull Request:

1. Add a buttons block, with different buttons in it, including outline.
2. Notice in the editor the outline button is black by default (it should be the accent color)
3. But in the front-end it's fine
4. Add a donate block, use a tiered layout with the modern style.
5. Switch to this branch and check the front-end and back-end.

### Other information:
